ui/src/views/scenes/Details.vue

```diff
@@ -301,7 +304,7 @@ export default {
     updatePlayer (src, projection) {
       this.player.reset()
 
-      const vr = this.player.vr({
+      /* const vr = */ this.player.vr({
```

Comment:
@lockedmouse why's that?

Reply:
ESLint shows a warning that `vr` is an unused variable. I didn't want to remove it totally because six lines below it is still used in commented-out code:

```js
// vr.camera.position.set(-1, 0, 2);
```

IMO this is the best compromise that removes the ESLint warning but preserves the code in case someone wants to uncomment the camera position. Personally, I am a proponent of deleting code over commenting it out, so alternatively I can just remove both comments if you want.